sass/compat/themes/buddyboss-theme/__imports.scss

```diff
@@ -0,0 +1,19 @@
+// ------------------------------------
+// DEFINE LOCAL VARIABLES
+// ------------------------------------
+$_body-page-selector:                  '.woocommerce-checkout' !default;
+$_body-theme-selector:                 '.theme-buddyboss-theme' !default;
+
+
+
+:root {
+	--fluidcheckout--field--height: #{ remsize( 4.4rem ) };
+	--fluidcheckout--field--border-radius: #{ remsize( .3rem ) };
```

Comment:
Field border-radius should be same as defined by the theme. BuddyBoss uses a CSS variable for that, so we need to use the same variable here too:

```
--fluidcheckout--field--border-radius: var( --bb-input-radius );
```

Reply:
Same as above.